components/mailbox/src/cypress.js

```diff
@@ -0,0 +1,46 @@
+function toggleComposer(event, component) {
```

Comment:
Adding this file brings back the side-effects problem we had before when our cypress tests targeted `index.html`. These should all be done only in the tests that require them, otherwise this adds this functionality to every test that loads `cypress.html` leading to unintended side-effects in each test making each test more difficult to debug.

Reply:
That's a good point. Will add this to tests. Thank for the suggestion, Kris!